### PR TITLE
Rename images from faas-netesd to faas-netes

### PR DESCRIPTION
**Pull request by mrtind**
This change builds images with name faas-netes instead of
faas-netes.

Signed-off-by: Martin Dekov (VMware) <mdekov@vmware.com>

Images are now builded - faas-netes.

## Description

Change faas-netes to build images with faas-netes instead of 
faas-netesd

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

It solves https://github.com/openfaas/faas-netes/issues/240.

## How Has This Been Tested?

Builded faas-netes locally with ```make build``` and ```make push``` into my own repo
which uploaded the image with ```latest``` tag.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by alexellis** [comment]
@mrtind I suggest you do a search and replace in the GitHub repo, there will be other instances of `faas-netesd` - i.e. in the YAML files - these have to be changed and tested too.

https://github.com/openfaas/faas-netes/search?q=faas-netesd&unscoped_q=faas-netesd

Alex

File: yaml_armhf/gateway.yml [hunk 1]
```diff
@@ -34,8 +34,8 @@ spec:
             memory: 100Mi
           limits:
             memory: 100Mi
-      - name: faas-netesd
-        image: functions/faas-netesd:0.5.2-armhf
+      - name: faas-netes
+        image: functions/faas-netes:0.5.2-armhf
```
Comment (alexellis) [inline yaml_armhf/gateway.yml]: We can't change this image yet..

File: yaml_arm64/gateway.yml [hunk 2]
```diff
@@ -34,8 +34,8 @@ spec:
             memory: 100Mi
           limits:
             memory: 100Mi
-      - name: faas-netesd
-        image: functions/faas-netesd:0.4.3-arm64
+      - name: faas-netes
+        image: functions/faas-netes:0.4.3-arm64
```
Comment (alexellis) [inline yaml_arm64/gateway.yml]: We can't change this image yet..

File: yaml/gateway-dep.yml [hunk 3]
```diff
@@ -54,8 +54,8 @@ spec:
             memory: 250Mi
           limits:
             memory: 250Mi   # Adjust to suit needs of deployment
-      - name: faas-netesd
-        image: openfaas/faas-netesd:0.5.7
+      - name: faas-netes
+        image: openfaas/faas-netes:0.5.7
```
Comment (alexellis) [inline yaml/gateway-dep.yml]: This image doesn't exist yet, I'll tag and push before merging.

**Comment by mrtind** [comment]
Fixed conflict through the github U.I. ...

**Comment by alexellis** [comment]
Please don't do that.. it won't work with the contribution process.

(Also the conflict is not cleared)

You'll need to reset this commit, add it again signed-off and then commit signed and do a force push.

Alex

**Pull request closed without merging**